src/components/header/SearchBar.js

```diff
@@ -49,48 +70,145 @@ const styles = theme => ({
     [theme.breakpoints.up('md')]: {
       width: 200,
     },
+    [theme.breakpoints.down('xs')]: {
+      paddingLeft: theme.spacing.unit * 4,
+    },
+  },
+  menuStyle: {
+    zIndex: 1,
+    minWidth: '200px',
+    maxHeight: '200px',
+    overflow: 'auto',
   },
 });
 
-export class SearchBar extends Component {
+class SearchBar extends Component {
   constructor(props) {
     super(props);
     this.state = {
       searchValue: '',
+      open: false,
+      shownEnv: modDisplay,
```

Comment:
consider changing this to something like `visibleEnvironmentCount`. More descriptive & explicit.

Reply:
done